pkg/backend/backend.go

```diff
@@ -18,8 +21,8 @@ import (
 )
 
 const (
-	maxWorkers      = 100
-	maxDequeueCount = 5
+	defaultMaxWorkers      = 20
```

Comment:
what's the rationale for decreasing this?

Reply:
Do you see more than `20` workers on one VM? 100 looks like a little bit overkill number? Or im overthinking?

Reply:
100 is fine, it can almost certainly be larger one day.  Most of the time the worker is just sitting waiting for someone else to do something - it does nothing memory intensive, and the only thing CPU intensive that it does is generate an SSH key.